src/main/java/org/sagebionetworks/bridge/spring/controllers/ParticipantController.java

```diff
@@ -602,6 +604,26 @@ public StatusMessage sendSmsMessageForWorker(@PathVariable String appId, @PathVa
         participantService.sendSmsMessage(app, userId, template);
         return new StatusMessage("Message sent.");
     }
+
+    @PostMapping("/v3/participants/emailRoster")
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    public StatusMessage getParticipantRosterForWorker() throws JsonProcessingException {
+        UserSession session = getAuthenticatedSession(WORKER);
+        String appId = session.getAppId();
+
+        StudyParticipant participant = session.getParticipant();
+        if (!participant.getEmailVerified() && !participant.getPhoneVerified()) {
+            throw new BadRequestException("Cannot request user data");
+        }
```

Comment:
We don't need to check phoneVerified. We should check that participant has an email address and that email verified is true.

Additional checks
* If the caller is a Study Coordinator but not a Researcher (use RequestContext.get().isInRole()), the caller must have an orgMembership (RequestContext.get().getCallerOrgMembership()).
* If the caller specifies a study and has org membership, we need to verify that RequestContext.get().getOrgSponsoredStudies() contains that study.